backend/Events/LobbyEvents.js

```diff
@@ -10,23 +10,56 @@ const MafiaGame = require('../domain/MafiaGame');
  * @param {MafiaGame} mafiaGame
  */
 function createLobby(io, socket, mafiaGame) {
-    socket.on('create-lobby', (createLobbyDTO) => {
-        console.log('New room request received');
-        // Create room and assign host player to the room
-        let roomID = mafiaGame.newGame();
-        let host = new Player(socket.id, roomID, createLobbyDTO.nickname);
-        mafiaGame.gameRoomsDict[roomID].addPlayer(host);
+  socket.on("create-lobby", (createLobbyDTO) => {
+    console.log("New room request received");
+    // Create room and assign host player to the room
+    let roomID = mafiaGame.newGame();
+    let host = new Player(socket.id, roomID, createLobbyDTO.nickname);
+    mafiaGame.gameRoomsDict[roomID].addPlayer(host);
+
+    // Subscribe to the room events
+    socket.join(roomID);
+
+    // Add player information to the host socket
+    socket.player = host;
+
+    // Send room ID back to host.
+    io.in(roomID).emit("lobby-code", new LobbyCodeDTO(roomID));
```

Comment:
Sends to everyone in the room, but should be fine since host will always be the only player in the room at this point